### ISSUE TITLE
[FR] add orderfile creation script to the NDK

### ISSUE THREAD

**Issue by DanAlbert** [android/ndk#1912]
### Description

Order files require some processing before they're consumed by lld (why? could we just upstream whatever the script is doing into lld?): https://android.googlesource.com/toolchain/pgo-profiles/+/refs/heads/main/scripts/create_orderfile.py. We should include that script in the NDK.

**Comment by appujee** [comment]
yeah, that's a good idea. if lld could incorporate these features, it'll make working with orderfile a lot easier. Currently there is no standard way to generate orderfile. the instrumentation based approach we are working on is just one of the ways.
